warehouse/static/js/warehouse/utils/analytics.js

```diff
@@ -1,53 +0,0 @@
-/* Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- * http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-/* global dataLayer */
-
-import * as cookie from "cookie";
-
-
-export default () => {
```

Comment:
Just a note that plausible already respects DNT and doesn't store PII, so an equivalent wrapper is not needed.